src/main/java/org/janelia/saalfeldlab/paintera/meshes/MeshManagerWithAssignmentForSegments.java

```diff
@@ -123,6 +118,7 @@
 
 	public MeshManagerWithAssignmentForSegments(
 			final DataSource<?, ?> source,
+			final LabelBlockLookup labelBlockLookup,
 			final Pair<? extends InterruptibleFunction<TLongHashSet, Interval[]>, Invalidate<TLongHashSet>>[] blockListCacheAndInvalidate,
```

Comment:
Passing the `labelBlockLookup` makes `blockListCacheAndInvalidate` obsolete, if I am not mistaken.

Reply:
For N5 backend yes, but currently `blockListCacheAndInvalidate` is a traditional `imglib2-cache` for `Label ID -> block lists` mapping, which is still helpful for example in case of file-based lookup where bulk requests cannot be performed more efficiently than individual queries. The new N5-specific cache allows to request a big bulk of block lists and store them which works behind the imglib2-based cache for individual entries.

I also find this logic a bit convoluted, but currently don't see a better way to put these two caches together if we want to be efficient and still support multiple label-to-block lookup formats.

Reply:
We could just add the imglib2-based cache into the individual implementations of `LabelBlockLookup` (or no cache at all if appropriate), no?